deploy/example.yaml

```diff
@@ -0,0 +1,207 @@
+    # should be created by default or not when OpenEBS gets
+    # installed.
+    sparse:
+      # enabled decides whether default cstor
+      # sparse pool should be configured as a part of openebs
+      # installation.
+      # If "true" a default cstor sparse pool will be configured, if
+      # "false" it will not be configured.
+      # This value takes effect only if createDefaultStorageConfig is
+      # set to true.
+      #
+      # Defaults to false
+      enabled: "false"
+    # nodeSelector specifies on which nodes the pods of this
+    # component can be scheduled.
+    #
+    # Default to nil i.e., all the nodes present in the cluster
+    # i.e., the pods can be scheduled anywhere in the cluster.
+    nodeSelector:
+    # tolerations can be used if we want the pods of this component
+    # to tolerate some of the taints of nodes.
+    #
+    # Default to nil
+    tolerations:
+
+  # provisioner stores the configuration for OpenEBS provisioner
+  provisioner:
+    enabled: "true"
+    imageTag:
+    replicas: 1
+    nodeSelector:
+    tolerations:
+
+  # localProvisioner stores the configuration for OpenEBS local
+  # provisioner
+  localProvisioner:
+    enabled: "true"
+    imageTag:
+    replicas: 1
+    nodeSelector:
+    tolerations:
+
+  # snapshotOperator stores the configuration for snapshot operator
+  snapshotOperator:
+    enabled: "true"
+    controller:
+      imageTag:
+    provisioner:
+      imageTag:
+    replicas: 1
+    nodeSelector:
+    tolerations:
+
+  # ndm stores the configuration for node-disk-manager
+  ndm:
+    enabled: "true"
+    imageTag: "v0.4.5"
+    replicas: 1
+    sparse:
+      path:
+      size:
+      count:
+    # filters contain the config for excluding or including vendors,
+    # paths, etc.
+    #
+    # Defaults for excluding vendors will be CLOUDBYT,OpenEBS, paths
+    # will be loop,fd0,sr0,/dev/ram,/dev/dm-,/dev/md
+    filters:
+      OSDisk:
+        enabled:
+        exclude: "/,/etc/hosts,/boot"
+      vendor:
+        enabled:
+        exclude: "CLOUDBYT,OpenEBS"
+      path:
+        enabled:
+        exclude: "loop,fd0,sr0,/dev/ram,/dev/dm-,/dev/md"
+        include: ""
+    # probes can be used to configure ndm probes such as Seachest,
+    # smart, capacity, etc.
+    probes:
+      enable:
+      - seachest
+    nodeSelector:
+    tolerations:
+
+  # ndmOperator stores the configuration for ndm operator
+  ndmOperator:
+    enabled: "true"
+    imageTag: "v0.4.4"
+    replicas: 1
+    nodeSelector:
+    tolerations:
+
+  # jiva stores the configuration for CASType: Jiva
+  jiva:
+    imageTag:
+    replicas: 1
+
+  # cstor stores the configuration for CASType: cstor
+  cstor:
+    pool:
+      imageTag:
+    poolMgmt:
+      imageTag:
+    target:
+      imageTag:
+    volumeMgmt:
+      imageTag:
+
+  admissionServer:
+    enabled:
+    imageTag:
+    failurePolicy:
+    replicas:
+    nodeSelector:
+    tolerations:
+    affinity:
+
+  helper:
```

Comment:
what is this?
Let us get appropriate name from OpenEBS team.
Is this jiva scrubber or something else.
We can have multiple fields if required even if they all use same image.